source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -7,28 +7,38 @@
 
 [WARNING]
 ====
-Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
+Do not let certificates expire. If they expire, the host and {hypervisor-shortname} stop responding, and recovery is an error-prone and time-consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
 ====
 
 .Procedure
 
-. In the Administration Portal, click menu:Compute[Hosts].
-. Migrate the virtual machines away from the host.
-. Click menu:Management[Maintenance] and btn:[OK].
-. Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
-. Once enrollment is complete, click menu:Management[Activate].
+. Renew the host certificates:
+.. In the Administration Portal, click menu:Compute[Hosts].
+.. Migrate the virtual machines away from the host.
```

Comment:
This step "Migrate..." needs to be checked. I think it is the same as putting the host in maintenance mode.

Reply:
so yes, let's say "click maintenance" and then say "VMs should automatically migrate away from the host. If they are pinned or otherwise non-migratable you need to shut them down."